New Bueno Logo

Lcjacobpd opened the pull request:
Made a new bueno logo for the main readme file

Commit:
New bueno logo

Updated the bueno logo & it's use in README.md

samuelkgutierrez reviewed (changes requested):
I like it! May I ask for a minor thing? Because of GitHub's dark mode it may be a little difficult to see. Can you add a lighter version (keep the one you already have) and link to that one in the README? Thank you.

Commit:
Bueno Logo Color Variants

+ switched to gray bueno logo in main readme
+ updated bueno.svg with other colors

samuelkgutierrez reviewed (approved):
Perfect. Thank you.
